package-lock.json

```diff
@@ -5167,6 +5175,12 @@
           "resolved": "https://registry.npmjs.org/regexp-tree/-/regexp-tree-0.1.13.tgz",
           "integrity": "sha512-hwdV/GQY5F8ReLZWO+W1SRoN5YfpOKY6852+tBFcma72DKBIcHjPRIlIvQN35bCOljuAfP2G2iB0FC/w236mUw==",
           "dev": true
+        },
+        "semver": {
```

Comment:
Can we remove these extra `package-lock.json` changes?

Reply:
I'll execute force push.